examples/devtools/package.json

```diff
@@ -34,16 +34,15 @@
 		"test:jest:coverage": "jest --detectOpenHandles --coverage --ci",
 		"tsc": "tsc",
 		"tsc:watch": "tsc --watch",
-		"watch:devtools-core": "tsc --watch --project ../devtools-core/tsconfig.json --preserveWatchOutput",
-		"watch:devtools-view": "tsc --watch --project ../devtools-view/tsconfig.json --preserveWatchOutput",
+		"watch:devtools-core": "tsc --watch --project ../../packages/tools/devtools/devtools-core/tsconfig.json --preserveWatchOutput",
+		"watch:devtools-view": "tsc --watch --project ../../packages/tools/devtools/devtools-view/tsconfig.json --preserveWatchOutput",
```

Comment:
This makes things feel funky to me. I get the motivation to move devtools-example, but I wonder if it's just misnamed and should remain where it is today. We used it as a local development tool to be able to iterate quickly when making changes to the extension (since it renders it inline with the app). That said, it's also caused issues because some things that work when the extension UI is rendered inline don't work when it is running as an actual extension installed in the browser (e.g. localStorage/sessionStorage are not shared by a running app and an installed extension, but the "inlined extension" can see the values set in the context of the app).

@Josmithr thoughts? I think it might be net good to move this out here and know that we _can_ use it for quick iteration, but also that we _must_ test any changes by actually installing the extension. And maybe update the npm scripts so `start` just starts the example app, and something like `start:watch` also does the watching of devtools/devtools-core for quick local iteration.

Reply:
Yeah, I think the naming is bad. It isn't really an example app, it's an integration test app. I would prefer to keep it where it is. We should probably just rename it.

Reply:
Restored it to the original location in latest.